confgenerator/confgenerator.go

```diff
@@ -242,6 +243,12 @@ func contains(s []string, str string) bool {
 
 	return false
 }
+func healthChecksLog(p platform.Platform) string {
+	if p.Type == platform.Windows {
+		return "${logs_dir}/health-checks.log"
+	}
+	return "${logs_dir}/../health-checks.log"
```

Comment:
Just wondering if there is a way to avoid the usage of `../` ? It may me confusing since `${logs_dir}` is different in the `google-ops-agent-engine`, `google-ops-agent-fluent-bit` and the windows services. I know it fixes the issue but wondering 🤔 .